controller/index.js

```diff
@@ -16,5 +16,32 @@ module.exports = {
           });
         });
     }
+  },
+  post: {
+    productImages: (shoeId, imageUrl) => {
+      return new Promise((resolve, reject) => {
+        model.postShoeImages(shoeId, imageUrl)
+        .then(res=>console.log('posted'))
+        .catch(err=>console.log(err))
+      })
+    }
+  },
```

Comment:
I see you passing resolve and reject to the promise constructor, but I don't see you using them anywhere in your promises.  Let's chat about this in OH

Reply:
Hi Michelle, noticed my postman was getting hung up on my update, post, and delete request. I figured out it was due to this resolve issue. I have updated my controller file to resolve and reject for each method. Will be included in the pull request I do tonight.